frontend/src/libs/components/chat-sidebar/chat-element/chat-element.tsx

```diff
@@ -0,0 +1,47 @@
+import { Icon } from '#libs/components/components.js';
+import { IconNameToIcon } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { useCallback } from '#libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  chat: Chat;
+};
+
+type Chat = {
+  id: number;
+  name: string;
+};
+
+// mocked data
+const selectedChat = {
+  id: 1,
+};
+
+const ChatElement: React.FC<Properties> = ({ chat }) => {
+  const handleSelectChat = useCallback(() => {
+    // TODO: redux logic for chat selection
+  }, []);
+  const isSelected = selectedChat.id === chat.id;
+
+  return (
+    <button
+      className={getValidClassNames(
+        styles['chat-element'],
+        isSelected && styles['selected'],
+      )}
+      onClick={handleSelectChat}
+    >
+      <div className={styles['name']}>
+        <img src="chat-logo.svg" alt="not found" />
```

Comment:
What about this icon?

Reply:
Working on it as we speak

Reply:
Don't forget about this icon

Reply:
Sorry, I missed it. 
I also wanted to ask: 
Do I have edit it myself? As far as I remember, chat-element was Yvhenii's ticket that he's been working on. This component was just showed here since I was pulling his Icon component. Not sure If I won't create conflicts messing with it.